specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2020-01-01/Servers.json

```diff
@@ -101,6 +101,58 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/servers/{serverName}/upgrade": {
```

Comment:
is serverVersion a property on the server resource? if yes, can this upgrade be enabled via a PUT/PATCH on the server resource itself? 
Because otherwise, the template scenario is broken. If i am using template for provisioning, to upgrade to a new version, i will to call this POST which cant be done from a template.

Reply:
This call does not just change a property, but posts an action to upgrade database server to a higher version, like upgrade mysql 5.6 to 5.7, changes engine and data format and so on, so I think it's better using POST